packages/shipit-cli/package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "chalk": "^2.4.1",
     "commander": "^2.15.0",
+    "lodash": "^4.17.10",
```

Comment:
Please don't add lodash just for `uniq`, add `lodash.uniq` instead or another package to make an array unique.

Reply:
Okay, I'm working on the tests and this issue now.

Reply:
I have added the tests for this new feature and also removed the lodash. @neoziro Can we close this PR soon, it has been a long time since I created this PR.